internal/configs/virtualserver.go

```diff
@@ -354,6 +366,23 @@ func generateUpstream(upstreamName string, upstream conf_v1alpha1.Upstream, isEx
 	}
 }
 
+func generateSlowStartForPlus(upstream conf_v1alpha1.Upstream, lbMethod string) string {
+
```

Comment:
the empty line is not necessary

Reply:
this wasn't addressed. we don't put empty lines at the beginning of the function